<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountingTAMBillPrinterGUI</class>
 <widget class="QDialog" name="AccountingTAMBillPrinterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stampa Lista su file ODT</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="billToPrintGroupBox">
       <property name="title">
        <string>Liste da stampare</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="1">
         <widget class="QComboBox" name="billToPrintComboBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="billToPrintLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Liste</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Documenti contabili</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QRadioButton" name="printBillButton">
          <property name="text">
           <string>Lista</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="printSummaryButton">
          <property name="text">
           <string>Sommario lista</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="printRawBillButton">
          <property name="text">
           <string>Brogliaccio lista</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="printAmountsOptionsGroupBox">
       <property name="title">
        <string>Stampa importi</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="3" column="0">
         <widget class="QRadioButton" name="printAllAmountsRadioButton">
          <property name="text">
           <string>Tutti &amp;gli importi</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="printTotalAmountsToDiscountRadioButton">
          <property name="text">
           <string>Importi al &amp;lordo del ribasso</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="printAmountsNotToDiscountRadioButton">
          <property name="text">
           <string>I&amp;mporti non soggetti a ribasso</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="printNoAmountsRadioButton">
          <property name="text">
           <string>&amp;Non stampare importi</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="printPriceItemOptionsGroupBox">
       <property name="title">
        <string>Descrizione voci di prezzo</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QRadioButton" name="printShortDescRadioButton">
          <property name="text">
           <string>Descrizione breve</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="printLongDescRadioButton">
          <property name="text">
           <string>Descrizione estesa</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="printShortLongDescRadioButton">
          <property name="text">
           <string>Descrizione breve ed estesa</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="printShortLongDescOptRadioButton">
          <property name="text">
           <string>Descrizione breve ed estesa (ottimizzato)</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="paperGroupBox">
       <property name="title">
        <string>Supporto</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="paperDimensionsGroupBox">
          <property name="title">
           <string>Dimensioni</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QComboBox" name="paperDimensionsComboBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="paperOrientationGroupBox">
          <property name="title">
           <string>Orientamento</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="0">
            <widget class="QRadioButton" name="paperHorizzontalRadioButton">
             <property name="text">
              <string>Ori&amp;zzontale</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="paperVerticalRadioButton">
             <property name="text">
              <string>&amp;Verticale</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>printShortDescRadioButton</tabstop>
  <tabstop>printLongDescRadioButton</tabstop>
  <tabstop>printShortLongDescRadioButton</tabstop>
  <tabstop>printShortLongDescOptRadioButton</tabstop>
  <tabstop>billToPrintComboBox</tabstop>
  <tabstop>paperDimensionsComboBox</tabstop>
  <tabstop>paperVerticalRadioButton</tabstop>
  <tabstop>paperHorizzontalRadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AccountingTAMBillPrinterGUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AccountingTAMBillPrinterGUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
